bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -36,14 +50,18 @@
       <div class="content-wrapper">
         <div class="phone">
           <div class="fxos-cta">
-            <h1><span>{{_('Firefox OS')}}</span></h1>
+            <h1>{{ high_res_img('img/firefox/os-new/fxos-wordmark.png', {'alt': _('Firefox OS'), 'width': '420', 'height': '85'}) }}</h1>
```

Comment:
Good point on the h1. Maybe SVG with a PNG fallback for IE8 and lower? Not worth the effort?